[22기_이수아] deploy 미션 제출합니다.

Immmii opened the pull request.

Immmii committed:
deploy: Dockerfile 추가

Immmii committed:
deploy: docker-compose

Immmii committed:
docs: readme update

Immmii committed:
chore: domain 패키지 구조 수정 (DDD)

dldusgh318 commented on docker-compose.yml:
혹시 레디스는 따로 사용하지않으셨나요??
만약 사용하신다면 레디스는 어떻게 띄우셨는지 궁금합니다!

dldusgh318 reviewed (commented):
수고하셨습니다!
대부분 네이밍을 변경하신 거라 배포부분관해서만 리뷰했습니다!

gilbert09031 reviewed (commented):
과제 수고하셨습니다 !
Actions에서 시도하신 워크플로우가 PR에는 포함되어있지 않네요...!

Immmii committed:
feat: OrderFood 결제 시스템 연동 및 동시성 문제 해결(Redisson Lock)

Immmii committed:
fix: orderNo 삭제

Immmii committed:
fix: Config 오류 수정

Immmii committed:
feat: ReservationMovie 결제 시스템 연동 및 동시성 문제 해결(Redisson Multi Lock)

Immmii committed:
fix(deploy): docker-compose mysql, redis 추가

Immmii committed:
chore: 파일 변경사항

Immmii committed:
deploy: Github Aciton workflows

Updated CI/CD workflow for CGV Clone project to include Docker Hub login, image building, and deployment steps.

Immmii committed:
fix(deploy): Github Aciton workflows

Immmii committed:
fix(deploy): Github Workflow

Working-directory 지정

Immmii committed:
fix(deploy): Github Aciton workflows 2
